pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -11590,30 +11635,32 @@ spec:
     resolver: %s
 `, resolverName))
 
-	cms := []*corev1.ConfigMap{
-		{
-			ObjectMeta: metav1.ObjectMeta{Name: config.GetFeatureFlagsConfigName(), Namespace: system.Namespace()},
-			Data: map[string]string{
-				"trusted-resources-verification-no-match-policy": config.FailNoMatchPolicy,
-			},
-		},
-	}
+			cms := []*corev1.ConfigMap{
```

Comment:
this code block is just move the previous test code into the for loop, and use the noMatchPolicy and verificationPolicies from the tests